fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java

```diff
@@ -100,6 +102,31 @@ public void setDependencies(@Nonnull Descriptors.FileDescriptor[] dependencies)
         this.dependencies = dependencies;
     }
 
+    /**
+     * Get local meta-data file descriptor.
+     */
+    @Nullable
+    public Descriptors.FileDescriptor getLocalFileDescriptor() {
+        return localFileDescriptor;
+    }
+
+    /**
+     * Set local meta-data file descriptor.
```

Comment:
```suggestion
     * Sets the local meta-data file descriptor.
```

Sorry, should have picked up this one earlier.